### DISCUSSION TITLE
Release 0.11.0-beta1

### DISCUSSION THREAD

**Discussion by FlorianLudwig** [Yakifo/amqtt#137] [Announcements]
Hi all,

the first release for `0.11.0` just dropped.

It's marked as beta as it needs some more testing.

If you are using anything more modern than python 3.7 - which I hope you do :) - you should definitely try it, as there is a major issue when running the broker on python 3.8 and newer!

**Comment by sivaNbalusu5** [comment]
@FlorianLudwig Is there any diff between 0.11.0 vs 0.11.0-beta1 when just using client? I mean no broker but just client to publish and subscribe to topics?

**Reply by FlorianLudwig** [reply]
@sivaNbalusu5 there is no `0.11.0` release, only `0.11.0-beta1` and `0.10.1`.

Between `0.10.1` the difference would be:

1) supported python versions
2) `0.11.x` has no hbmqtt compatiblity layer anymore

**Comment by sivaNbalusu5** [comment]
@FlorianLudwig Looks good. Again sorry for the confusion, I meant to ask difference between 0.10.1 and 0.11.0-beta1 for clients.

**Comment by sjlongland** [comment]
I gave it a try today… under Python 3.10 it seems to work okay, but under 3.11, WebSockets transport is broken.

I then disappeared down a rabbit hole trying to get it working, decided to shelve it for now.

https://github.com/sjlongland/amqtt/tree/feature/py311-bump-dependencies is my branch…

- `poetry` 1.4 and later won't install the dev dependencies because the syntax changed. (https://github.com/sjlongland/amqtt/commit/3120987de0cbfd3a8379290aede68f2e1e9d6ffe)
- `mypy` had to be bumped (I picked 1.3.0) because the existing version would not install (`ast27/Custom/../Include/compile.h:5:10: fatal error: code.h: No such file or directory`; https://github.com/sjlongland/amqtt/commit/395c2a6de277dc0b1c177d9c067a37e35024264a)
- `hypothesis` fails to import with `FileNotFoundError: Can't open orphan path` -- tried bumping that, but it didn't help and I'm not sure why it's barfing.

I'll do some more testing under Python 3.10 for now and re-visit Python 3.11 in a few weeks.

**Comment by ajmirsky** [comment]
amqtt now supports python 3.10+ and regression testing is done against 3.10-3.13.
